### PR TITLE
Fix style in //flow/instrumentation

### PR DESCRIPTION
**Pull request by abarth**
This patch makes these files better match Google C++ style.

Also, add the engine lap time when running on Mojo.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @chinmaygarde

File: sky/shell/gpu/mojo/rasterizer_mojo.cc [hunk 1]
```diff
@@ -82,6 +82,8 @@ void RasterizerMojo::Draw(uint64_t layer_tree_ptr,
     return;
   }
 
+  paint_context_.engine_time().SetLapTime(layer_tree->construction_time());
```
Comment (Hixie) [inline sky/shell/gpu/mojo/rasterizer_mojo.cc]: Does this fix the problem where on mojo builds, the CPU graph is always blank?

**Comment by Hixie** [comment]
LGTM

File: sky/shell/gpu/mojo/rasterizer_mojo.cc (hunk 1 above)
Comment (abarth) [inline sky/shell/gpu/mojo/rasterizer_mojo.cc]: Yes.  Is there a bug on file for that?

Comment (Hixie) [inline sky/shell/gpu/mojo/rasterizer_mojo.cc]: I don't think so, but @apwilson keeps hitting it, so he'll be happy.

**Pull request merged**